data/pg/ACCESSCONTROL-ACTIONS-TEST/actions-test.json

```diff
@@ -203,7 +203,7 @@
       "enabled": true,
       "serviceCode": "ESTIMATE",
       "code": "null",
-      "navigationURL": "/works-ui/employee/estimate/search-estimate",
+      "navigationURL": "/works-ui/employee/estimate/search-estimate?status=APPROVED",
```

Comment:
It shouldn't be added for search estimate
It should be added at create contract 
      "navigationURL": "/works-ui/employee/estimate/search-estimate",